src/astra_rl/ext/transformers/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .hf_ast_problem import HFASTProblem
```

Comment:
What's the thinking behind the hierarchy of `ext/transformers/`?

Would something like `astra_rl/ext/huggingface.py` or `astra_rl/problems/huggingface.py` be more intuitive?

`transformers` seems to be a very-package-specific/implementation grouping that seems a little misaligned for storing HFProblem to me.

Reply:
This is very fair! However, transformers does [not yet](https://discuss.huggingface.co/t/static-type-checking-with-mypy-whats-the-official-position/464/4) support typed Python, so I wanted to isolate the unpureness to a separate folder. This is noted in the "sharp edges" above.